config/initializers/bgs.rb

```diff
@@ -8,6 +8,8 @@
   config.client_station_id = Settings.bgs.client_station_id
   config.client_username = Settings.bgs.client_username
   config.env = Rails.env.to_s
+  config.external_uuid = Settings.bgs.external_uuid
```

Comment:
Might need to add these to [Jenkins](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-api/detail/add_bgs_external_key_info/1/pipeline) somehow